libs/perun/components/src/lib/groups-list/groups-list.component.html

```diff
@@ -62,6 +60,7 @@
       <tr
         mat-row
         *matRowDef="let group; columns: displayedColumns;"
+        [routerLink]="disableRouting ? null : ['/organizations', group.voId, 'groups', group.id]"
         class="dark-hover-list-item">
```

Comment:
Please, duplicate the `tr` element, where one instance has the routerLink set and the other one doesn't. Then by using the NgIf decide, which one to show. Because with this implementation, if the routing is disabled, the element is still "clickable" and it doesn't look good.